segway/gmtk/params.py

```diff
@@ -317,6 +419,31 @@ def __str__(self) -> str:
         return " ".join([self.mean, self.covar])
 
 
+class MissingFeatureDiagGaussianMC(MC, object):
+    """
+    Attributes:
+        component_type = "COMPONENT_TYPE_MISSING_FEATURE_SCALED_DIAG_GAUSSIAN"
+        mean: str: name of Mean object associated to this MC
+        covar: str: name of Covar obejct associated to this MC
+    """
+    def __init__(self, mean: str, covar: str):
+        """
+        Initialize a single MissingFeatureDiagGaussianMC object.
+        :param mean: name of Mean object associated to this MC
+        :param covar: name of Covar obejct associated to this MC
+        """
+        # more component types?
```

Comment:
I don't understand this comment. We aren't implementing more so you can delete it

Reply:
Removed.